docs/tutorials/mask_maps.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Create an exclusion mask\n",
+    "# Mask maps\n",
     "\n",
     "## Introduction\n",
```

Comment:
If we follow the **Prerequisites** , **Context** **Proposed approach**, structure we decided upon, eg as here https://docs.gammapy.org/dev/tutorials/light_curve_flare.html then **Introduction** is not necessary.

I think the *Objective* here is also much broader now than just making an exclusion mask around the Crab, so that can be modified as well.